spec/card_spec.rb

```diff
@@ -1,4 +1,4 @@
-require './lib/card'
+require './lib/card.rb'
```

Comment:
You can just use `./lib/card`, you should not need an `.rb`. in these require statements.